css/layout.css

```diff
@@ -0,0 +1,139 @@
+.hidden {
+  display: none;
+}
+
+.container {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+  text-align: center;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-half-h {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.input-bar {
+  width: 70%;
+  height: 35%;
+}
+
+.search-button {
+  width: 22%;
+  height: 35%;
+}
+
+.button-row {
+  display: flex;
+  justify-content: space-evenly;
+  margin-bottom: 10px;
+}
+
+.top-rank,
+.top-genre,
+.random-rank,
+.random-genre {
+  display: none;
+}
```

Comment:
Instead of having a rule for this, you should just add and remove the `hidden` class you created above to these elements.

Reply:
These elements are hidden while the screen size is less than 768px, but if a user widens his screen they appear. 
Would I use JavaScript to check the screen size and change their class name if the screen is less than 768px?
If so,  would I keep checking to see a screen change?